packages/valory/skills/dynamic_nft_abci/handlers.py

```diff
@@ -291,15 +294,19 @@ def _handle_get_health(
             )  # this can be negative if we have passed the estimated reset time without resetting
             is_healthy = seconds_since_last_reset < 2 * observation_interval
```

Comment:
Sorry let me articulate that better;

our overarching healthy state should be a all([
   seconds_since_last_tm_update within tolerance,
   seconds since last reset > 2*obervation_interval
])

Reply:
Ok, added now